product_docs/docs/pgd/5/durability/camo.mdx

```diff
@@ -131,59 +133,61 @@ CAMO relies on a retry loop and specific error handling on the client side. Ther
 
-To illustrate this, we have an example of a retry loop in a CAMO-aware client application, written in a C-like pseudo-code. It expects two DSNs `origin_dsn` and `partner_dsn` providing connection information. These usually are the same DSNs as used for the initial call to `bdr.create_node` and can be looked up in `bdr.node_summary`, column `interface_connstr`.
+To illustrate this, this example shows a retry loop in a CAMO-aware client application, written in a C-like pseudo-code. It expects two DSNs, `origin_dsn` and `partner_dsn`, providing connection information. These usually are the same DSNs as used for the initial call to `bdr.create_node` and can be looked up in `bdr.node_summary`, column `interface_connstr`.
 
 ```
 PGconn *conn = PQconnectdb(origin_dsn);
 ```
 
-The process starts connecting to the origin node. We now enter our loop:
+The process starts connecting to the origin node. Now enter the loop:
 
 ```
 loop {
     PQexec(conn, "BEGIN");
 ```
 
-We start our transaction and begin populating it with changes:
+Next, start the transaction and begin populating it with changes:
 
 ```
     PQexec(conn, "INSERT INTO ...");
     ...
 ```
 
-Once we are done, we need to make a record of the local node id and the transaction id. Both are available as parameters.
+Once you're done, you need to make a record of the local node id and the transaction id. Both are available as parameters.
 
 ```
     node_id = PQparameterStatus(conn, "bdr.local_node_id");
     xid = PQparameterStatus(conn, "transaction_id");
 ```
 
-We are now ready to try and commit.
+Now it's ready to try to commit.
 
 ```
     PQexec(conn, "COMMIT");
     if (PQresultStatus(res) == PGRES_COMMAND_OK)
         return SUCCESS;
 ```
 
-If the result is `PGRES_COMMAND_OK`, we are good and ready to move on. But if it isn't, we need to use CAMO to track the transaction to completion. The first question we need to ask is was the connection bad?
+If the result is `PGRES_COMMAND_OK`, that's good, and you can move on. But if it isn't, you need to use CAMO to track the transaction to completion. The first question to ask is, "Was the connection bad?"
 
 ```
     else if (PQstatus(res) == CONNECTION_BAD)
     {
 ```
 
-If it was a bad connection, then we can check on the CAMO partner node to see if the transaction made it there.
+If it was a bad connection, then you can check on the CAMO partner node to see if the transaction made it there.
 
 ```
         conn = PQconnectdb(partner_dsn);
         if (!connectionEstablished())
             panic();
 ```
 
-If we can't connect to the partner node, there's not a lot we can do. So we should panic, or take similar actions.
+If you can't connect to the partner node, there's not a lot you can do. So we should panic, or take similar actions.
```

Comment:
I don't think "panic" is a technical term? I needed to change this to second person, but I wasn't sure if this was humor (which we shouldn't use) or what.

Reply:
It is - panic means to dump core and exit with an error. for example in Go language it is a standard function to panic when an error condition can't be resolved.

Reply:
```suggestion
If you can't connect to the partner node, there's not a lot you can do. In this case, panic, or take similar actions.
```